app.py

```diff
@@ -5,7 +5,9 @@
 from error_handling import CustomError, ValidationError, WrongIdError
 
 app = Flask(__name__)
-with open("restaurants.json", "r") as read_file:
+
+app.config.from_object('config.Config')
```

Comment:
the thing about configuration that it is changeable, configurable depending on the environment, while here config to use is hardcoded.
it should be like `if Dev environment, then use DevConfig; if Prod environment, use ProdConfig...`
for example, you can use `FLASK_ENV` to decided which env to use

Reply:
also, you may be interested in predefined Flask configuration variables:
https://flask.palletsprojects.com/en/1.1.x/config/#builtin-configuration-values

Reply:
fixed